micro-rdk/src/common/conn/viam.rs

```diff
@@ -523,6 +523,23 @@ where
             {}
         }
 
+        // Initiate SNTP
+        #[cfg(feature = "esp32")]
+        unsafe {
+            use crate::esp32::esp_idf_svc::sys;
+            log::info!("initializing SNTP service");
+            sys::esp_sntp_setoperatingmode(sys::esp_sntp_operatingmode_t_ESP_SNTP_OPMODE_POLL);
+            let ntp_addr = std::ffi::CString::new("pool.ntp.org").unwrap();
+            sys::esp_sntp_setservername(0, ntp_addr.as_ptr());
```

Comment:
Why not using the esp idf svc driver?

Reply:
Because I somehow missed the fact there was one and went straight to the c bindings 🙃.

Will rework this to use the rust driver

Reply:
reworked